### DISCUSSION TITLE
is there a way to reduce image density (DPI) but keep the geometry (print size)?

### DISCUSSION THREAD

**Discussion by braindevices** [ImageMagick/ImageMagick#5552] [Help] [answered]
I have png files. I would like the resample it to lower DPI. I would expect the `-resample` will do this. However, `mogrify -units PixelsPerINch -density 150`  won't help, it just change the resolution field instead of re-sample the image. Resample also does not work `mogrify -units PixelsPerINch -resample 150x150`, it has exact same effect as `-density`.

I would like to change `1000x1000pixel 100DPI` image to `100x100pixel 10DPI` image, so the image print size remains the same. How can I do this?

**Comment by fmw42** [comment, marked as answer]
See -resample at https://imagemagick.org/script/command-line-options.php#resample

**Reply by braindevices** [reply]
@fmw42 please check my update. I am aware of resample. It simply does not work as expected.

**Reply by braindevices** [reply]
wait, it could be I overwritten the original fille.

**Reply by braindevices** [reply]
sorry, my first `-density` actually modified the original files ... so the `-resample` had no effect. `-resample` is actually the correct way.
